requirements.txt

```diff
@@ -1,2 +1,4 @@
+git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
Very very personal opinion here: locking the version surely is a good thing, but I think it should be done in a neat way. It's a good point you raise here, I don't think we've pondered on it in this context… Anyway: a neat way would be to use GitHug tags, or (I prefer) uploading it to PyPI.